modules/virt-creating-configmap.adoc

```diff
@@ -167,103 +242,37 @@ metadata:
   name: os-configmap
   namespace: default <1>
 data:
-  guestos2common: | <2>
-    "Red Hat Enterprise Linux Server": "rhel"
-    "CentOS Linux": "centos"
-    "Fedora": "fedora"
-    "Ubuntu": "ubuntu"
-    "openSUSE": "opensuse"
-  osinfo2common: | <3>
-    "rhel_6_9_plus_ppc64": "rhel6.9"
-    "rhel_6_ppc64": "rhel6.9"
-    "rhel_6": "rhel6.9"
-    "rhel_6x64": "rhel6.9"
-    "rhel_7_ppc64": "rhel7.7"
-    "rhel_7_s390x": "rhel7.7"
-    "rhel_7x64": "rhel7.7"
-    "rhel_8x64": "rhel8.1"
-    "sles_11_ppc64": "opensuse15.0"
-    "sles_11": "opensuse15.0"
-    "sles_12_s390x": "opensuse15.0"
-    "ubuntu_12_04": "ubuntu18.04"
-    "ubuntu_12_10": "ubuntu18.04"
-    "ubuntu_13_04": "ubuntu18.04"
-    "ubuntu_13_10": "ubuntu18.04"
-    "ubuntu_14_04_ppc64": "ubuntu18.04"
-    "ubuntu_14_04": "ubuntu18.04"
-    "ubuntu_16_04_s390x": "ubuntu18.04"
-    "other_linux": "fedora31"
-    "windows_10x64": "windows"
-    "windows_2008x64": "windows"
-    "windows_2016x64": "windows"
+  osinfo2common: |
+    "<windows-version>": "windows" <2>
 EOF
 ----
-<1> Optional: You can update the namespace.
-<2> `guestos2common` maps the operating system reported by the guest agent to a corresponding {VirtProductName} operating system.
-<3> `osinfo2common` maps the RHV VM operating system designation to an {VirtProductName} template.
-// Windows templates to re-add when Win bug is BZ1856654 is fixed, if we still need custom configmaps
-// "windows_10": "win10"
-// "windows_10x64": "win10"
-// "windows_2003": "win10"
-// "windows_2003x64": "win10"
-// "windows_2008R2x64": "win2k8"
-// "windows_2008": "win2k8"
-// "windows_2008x64": "win2k8"
-// "windows_2012R2x64": "win2k12r2"
-// "windows_2012x64": "win2k12r2"
-// "windows_2016x64": "win2k16"
-// "windows_2019x64": "win2k19"
-// "windows_7": "win10"
-// "windows_7x64": "win10"
-// "windows_8": "win10"
-// "windows_8x64": "win10"
-// "windows_xp": "win10"
+<1> Optional: You can change the value of the `namespace` parameter.
+<2> Specify your Windows version, for example, `"windows_2016x64"`. All Windows versions are mapped to the Windows template you created.
```

Comment:
Specify your Windows version, as it appears in RHV Rest api, for example, `"windows_2016x64"`. 
All Windows versions should be mapped to the Windows custom template that was created in previous step.